backend/tests/test_auth.py

```
@@ -3,23 +3,22 @@
+from database.session import SessionLocal, engine
 from models.user import User
 from core.security import hash_password
-from sqlalchemy.orm import Session
 
 # Initialize FastAPI test client
-client = TestClient(app)
-
-# Ensure tables exist before testing
-Base.metadata.create_all(bind=engine)
+@pytest.fixture
+def client():
```

Ruff (0.8.2)

Missing return type annotation for public function `client`

(ANN201)